### ISSUE TITLE
Home - Learn more color contrast issue - button style link

### ISSUE THREAD

**Issue by iamjolly** [18F/samhsa-prototype#299]
### Problem Description
The Learn More button-style link on the home page uses blue (#1A7BBC) on top of a gray (#f0f0f0) background. The contrast ratio is 4.19:1, which fails WCAG 2.0 AA (should be 4.5:1) for this size of text.

### Solution
The blue against white passes the 4.5:1 threshold, so consider making the button's background color white instead of inheriting the gray color of the `div` with the classes `Home___StyledDiv5-bwapfq-11 doBRzF`. 
Alternatively, the blue can be made darker, but it may be inconsistent with the widespread use of #1A7BBC color on the site. 

![Screenshot of color contrast check in Chrome](https://user-images.githubusercontent.com/1093423/64656824-f1443600-d3ed-11e9-87be-fa47af3e05cd.png)

**Comment by iamjolly** [comment]
This is now showing up with a white background for the area around the button, so this issue can be closed unless the gray background is still desired.

**Comment by allisonnorman** [comment]
Thanks, Robert!
